content/documentation/staging/configuration/authentication/openid.adoc

```diff
@@ -48,19 +48,25 @@ authenticated requests to the Kubernetes API.
 
 == Set-up
 
-If you want to enable usage of the OpenId's _authorization code flow_, make sure that
-the Kiali's signing key is 16, 24 or 32 bytes long. If you install Kiali via
-the operator and don't set a custom signing key, the operator should create a
-16 bytes long signing key. If you setup a signing key of a different size,
-Kiali will only be capable of using the _implicit flow_. 
+If you want to enable usage of the OpenId's _authorization code flow_, make
+sure that the
+link:https://github.com/kiali/kiali-operator/blob/7dafc469c95d4307ebd03c515a87c7f84eb64da7/deploy/kiali/kiali_cr.yaml#L746-L754[Kiali's
+signing key] is 16, 24 or 32 bytes long. If you install Kiali via the operator
+and don't set a custom signing key, the operator should create a 16 bytes long
+signing key. If you setup a signing key of a different size, Kiali will only be
+capable of using the _implicit flow_. 
```

Comment:
This is a little bit confusing.  I think the last sentence implies that if you set up a key with size no in [16,24,32] that you can only use implicit flow.   But because it is after the sentence about Kiali defining a 16 byte long it makes me think maybe I need to define only a 16 byte long key.  I would suggest reversing the order of the last two sentences. Also, it should be "16 byte long" as opposed to "16 bytes long".